tests/e2e/cypress/cypress/support/api.js

```diff
@@ -0,0 +1,75 @@
+/*
+
+RERO ILS
+Copyright (C) 2020 RERO
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, version 3 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+/** API call to logout method ============================================== */
+Cypress.Commands.add('api_logout', () => {
```

Comment:
Can we keep a camelCase for commands designation as done in other commands? -> `apiLogout`